lib/offline/storage.js

```diff
@@ -561,11 +576,6 @@ shaka.offline.Storage = class {
       await muxer.init();
       activeHandle = await muxer.getActive();
 
-      // Store the data.
-      const dataKeys = await activeHandle.cell.addSegments([data]);
-      dataKey = dataKeys[0];
-      throwIfAbortedFn();
-
       // Acquire the mutex before accessing the manifest, since there could be
       // multiple instances of this method running at once.
       mutexId = await shaka.offline.Storage.mutex_.acquire();
```

Comment:
I don't think we need this mutex anymore? Since any given store operation is only going to be running this method once or twice anyway, and in the two-calls case they definitely don't happen at the same time. So there's no risk of a given storage entry being mutated by two different threads at once.

And if we get rid of this mutex, we could get rid of the `shaka.util.Mutex` class too.

Reply:
Removed Mutex.